libs/island-ui/core/src/lib/RadioButton/RadioButton.tsx

```diff
@@ -109,25 +114,28 @@ export const RadioButton = ({
             </Text>
           )}
         </span>
+        {large && Illustration && (
+          <Box marginLeft="auto" paddingRight="smallGutter">
+            <Illustration />
+          </Box>
+        )}
         {tooltip && (
           <div
             className={cn(styles.tooltipContainer, {
               [styles.tooltipLargeContainer]: large,
+              [styles.toolTipLargeContainerWithIllustration]:
+                large && Illustration,
             })}
```

Comment:
this should probably be separated so no styles are overwritten, if large is true and/or Illustration is true marginLeft will conflict between all classnames, this makes the styles more fragile